src/components/camera/Client.ts

```diff
@@ -44,9 +44,9 @@ export class CameraClient implements Camera {
     return response.getImage_asU8();
   }
 
-  async renderFrame(mimeType: MimeType) {
+  async renderFrame(mimeType: MimeType = '') {
     const cameraService = this.cameraService;
-    const request = new pb.GetPointCloudRequest();
+    const request = new pb.RenderFrameRequest();
```

Comment:
I'm a little confused as to how this was working... I guess the request signatures are similar enough?